res/values-in/strings.xml

```diff
@@ -53,8 +53,8 @@
     <string name="SERVER">"SERVER"</string>
     <string name="Joining_game_in_">"Bergabung permainan di "</string>
     <string name="Game_Full">"Permainan Penuh"</string>
-    <string name="US_East">"Amerika Tengah"</string>
-    <string name="US_West">"Amerika Utara"</string>
+    <string name="US_East">"AS Timur"</string>
+    <string name="US_West">"AS Barat"</string>
```

Comment:
Previous texts at lines 56 and 57 were already correct (because in english is Central America and North America). Please return them.